src/Markdownify/Converter.php

```diff
@@ -806,16 +807,21 @@ protected function handleTag_a_converter($tag, $buffer)
             return '[' . $buffer . '](' . $this->getLinkReference($tag) . ')';
         }
 
-        // [This link][id]
-        foreach ($this->footnotes as $tag2) {
-            if ($tag2['href'] == $tag['href'] && $tag2['title'] === $tag['title']) {
-                $tag['linkID'] = $tag2['linkID'];
-                break;
+        if ($this->$linkPosition == self::LINK_IN_LINE){
```

Comment:
Attention! The tests are broken because of the `$` before `linkPosition` :dancers: